### ISSUE TITLE
Archive links not accessible to keyboard users

### ISSUE THREAD

**Issue by stevefaulkner** [gitterHQ/gitter#784]
When I view https://gitter.im/w3c/a11ySlackers/archives/all and try to navigate to the 'linked' daily archives using the keyboard I cannot. This makes it impossible for keyboard users to access the archives.
